esp/esp/program/modules/handlers/teachereventsmodule.py

```diff
@@ -195,7 +195,7 @@ def teacher_events(self, request, tl, one, two, module, extra, prog):
         training_times = self.getTimes('training')
         
         for ts in list( interview_times ) + list( training_times ):
-            ts.teachers = [ x.user.first_name + ' ' + x.user.last_name + ' <' + x.user.email + '>' for x in self.entriesBySlot( ts ) ]
```

Comment:
I don't like losing the email addresses in this.  I think the intention is that you can just copy-paste the entire block into an email to field, and send to all teachers with a given teacher training.  So it would be nice to keep that, possibly by having an option to the userlist tag that lets you also display email addresses.